test/prompConfig.js

```diff
@@ -1,11 +1,32 @@
+import fs from 'fs-extra';
 import test from 'ava';
 import path from 'path';
 import chalk from 'chalk';
 import json from 'json-extra';
+import os from 'os';
```

Comment:
I would suggest to import `fs-extra` after `chalk` and `os` as the first. to keep the import statement structure more clear.